docs/Developers-Guide.md

```diff
@@ -1,9 +1,21 @@
 # Developers Guide
 
+## Setup OTP
+_A Quick guide to setting up the OpenTripPlanner project._
+
+You need Git, Maven and Java(JDK) and an IDE installed on your computer.
+
+1. Clone OpenTripPlanner from GitHub.
+2. Checkout the desired branch `git checkout dev-2.x` 
+3. Run `maven package`- this will download all dependencies, build the project and run tests.
```

Comment:
No command line Maven build is needed to import the project into an IDE. Generally the IDE Maven build and command line build are quite separate and I think the IDE even has its own artifact cache.

Reply:
I prefer to be able to run Maven from the command line. Running it in the IDE is error prune and often do not give the same outcome. If it does not work (problems downloading or file access) the failure will be much more visible than in the IDE. Before submitting a PR, I expect a developer to run `maven clean package` locally - to verify that tests run. Also, this is a Quick Guide - for the developer that know what she is doing, he may skip the rest of the document. The next chapter is how to set up OTP with an IDE.